### PR TITLE
Ek/contact page

### PR DESCRIPTION
**Pull request by EmiKusaba**
I need to see more inspirations. but It was great learn.

### CODE REVIEW

File: src/contact/contact.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```
Comment (juan-rod) [inline src/contact/contact.html]: forgot to mention this on the about page. move these script tags to the bottom of the body. right before `</body>`

File: src/contact/contact.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      <a href="#"></a>
+      <img src="./img/instagram.png" alt="imstagram" class="icon">
+    </div>
+    <div class="icon">
+      <a href="#"></a>
+      <img src="./img/linkedin.png" alt="linkedin" class="icon">
+    </div>
+  </div>
+</section>
+
```
Comment (juan-rod) [inline src/contact/contact.html:100]: put the script tags in your `head` here!

File: src/contact/style.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  border: 1px solid gray;
+}
+
+label {
+  display: block;
+  margin-bottom: 0;
+}
+
+sup {
+  color: red;
```
Comment (juan-rod) [inline src/contact/style.css]: this read is very loud. what if we muted it a bit by playing with the alpha. try `color: rgba(255, 0, 0, 0.71);`

**Review by juan-rod — APPROVED** [review]
great work! review the suggestions and merge!

File: src/contact/style.css (hunk 3 above)
Comment (EmiKusaba) [reply src/contact/style.css]: Looks better now! Thank you very much!

**Comment by EmiKusaba** [comment]
Thanks, I will move them!

Can I ask what is the difference between putting them at the end of body and putting them in head?

**Pull request merged**